traffic_ops/traffic_ops_golang/profileparameter/postprofileparameter.go

```diff
@@ -48,6 +51,13 @@ func PostProfileParam(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("posting profile parameter: "+err.Error()))
 		return
 	}
+	profileName, ok, err := dbhelpers.GetProfileNameFromID(inf.Tx.Tx, *profileParam.ProfileID)
+	if err != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("getting profile name from id: "+err.Error()))
+	} else if !ok {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusBadRequest, nil, errors.New("profile not found"))
```

Comment:
Shouldn't this be using `http.StatusNotFound`? Also, 'parameter not found' might be useful for the user to read, so that they know it failed because the _profile_ wasn't found, not because the _parameter_ wasn't found.